### DISCUSSION TITLE
Welcome to Django-ChatBot Discussions!

### DISCUSSION THREAD

**Discussion by PranavDalvi** [PranavDalvi/Aapi-Ai-Django-ChatBot#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RavinderSingh13-git** [comment]
Hello Pranav and Team, Greetings!!

First of all KUDOS for your GREAT work on this project. I am a beginner in Django and Python and was trying to run your project(later will understand also line by line code). I have few questions here; I have windows in my laptop and in VSCode using venv environment I have installed all mentioned packages(with exact same versions mentioned in your requirements.txt file) in requirements.txt but when I do migrations(running your project with my secret key from a test Django project) I am getting lot of errors. I am attaching the file here. 

Would like to request you to please do help me, I really want to learn coding but are on my own, then saw this GREAT project for learning, will be grateful for your help and inputs. THANK YOU again for sharing such a nice Project.

[Errors_git_Django_project.txt](https://github.com/PranavDalvi/Aapi-Ai-Django-ChatBot/files/10453076/Errors_git_Django_project.txt)

**Comment by RavinderSingh13-git** [comment]
BINGO, I have spent quite few hours on this one and I have to install 2 packages in my virtual environment to get it running fine.

Following are the 2 packages we need to install to avoid 500 internal error in this project.

```
pip install -U scikit-learn scipy matplotlib
pip install tensorflow-gpu
```
